plugin/trino-prometheus/src/main/java/io/trino/plugin/prometheus/PrometheusConnectorConfig.java

```diff
@@ -35,6 +35,7 @@
     private Duration queryChunkSizeDuration = new Duration(1, TimeUnit.DAYS);
     private Duration maxQueryRangeDuration = new Duration(21, TimeUnit.DAYS);
     private Duration cacheDuration = new Duration(30, TimeUnit.SECONDS);
+    private Duration prometheusReadTimeout = new Duration(10, TimeUnit.SECONDS);
```

Comment:
The prefix `prometheus` is redundant. Let's rename to `readTimeout` including the getter and setter.